channels/serializers.py

```diff
@@ -58,20 +75,34 @@ def get_user_is_moderator(self, channel):
         """
         return bool(channel.user_is_moderator)
 
+    def get_membership_is_managed(self, channel):
+        """
+        Get membership_is_managed from the associated Channel model
+        """
+        try:
+            channel_obj = Channel.objects.get(name=channel.display_name)
```

Comment:
I've got an older channel where name != display_name, so this line raises the DoesNotExist exception and returns `False` for `membership_is_managed` even though the actual value is `True`; is this the intended behavior?

Reply:
As far as I know `name` should always be the same as `display_name`. When you have a moment can you show me the channel in person?